addons/medical_engine/XEH_postInit.sqf

```diff
@@ -1,93 +1,116 @@
-["unit", {
-    params ["_new"];
-    [_new] call FUNC(updateDamageEffects); // Run on new controlled unit to update QGVAR(aimFracture)
-}, true] call CBA_fnc_addPlayerEventHandler;
+["CBA_settingsInitialized", {
+    if !(GETEGVAR(medical,enabled,false)) exitWith {};
 
-["CAManBase", "init", {
-    params ["_unit"];
+    [QGVAR(updateDamageEffects), LINKFUNC(updateDamageEffects)] call CBA_fnc_addEventHandler;
 
-    if (unitIsUAV _unit) exitWith {TRACE_1("ignore UAV AI",typeOf _unit);};
-    if (getNumber (configOf _unit >> "isPlayableLogic") == 1) exitWith {TRACE_1("ignore logic unit",typeOf _unit);};
+    ["unit", {
+        params ["_new"];
+        [_new] call FUNC(updateDamageEffects); // Run on new controlled unit to update QGVAR(aimFracture)
+    }, true] call CBA_fnc_addPlayerEventHandler;
 
-    private _allHitPoints = getAllHitPointsDamage _unit param [0, []];
-    if ((GVAR(customHitpoints) arrayIntersect _allHitPoints) isNotEqualTo GVAR(customHitpoints)) exitWith {
-        ERROR_1("Bad hitpoints for unit type ""%1""",typeOf _unit);
-    };
+    ["CAManBase", "init", {
+        params ["_unit"];
 
-    // Calling this function inside curly brackets allows the usage of
-    // "exitWith", which would be broken with "HandleDamage" otherwise.
-    _unit setVariable [
-        QEGVAR(medical,HandleDamageEHID),
-        _unit addEventHandler ["HandleDamage", {_this call FUNC(handleDamage)}]
-    ];
-}, nil, [IGNORE_BASE_UAVPILOTS], true] call CBA_fnc_addClassEventHandler;
-
-#ifdef DEBUG_MODE_FULL
-[QEGVAR(medical,woundReceived), {
-    params ["_unit", "_damages", "_shooter", "_ammo"];
-    TRACE_4("wound",_unit,_damages,_shooter,_ammo);
-    //systemChat str _this;
-}] call CBA_fnc_addEventHandler;
-#endif
+        if (unitIsUAV _unit) exitWith {TRACE_1("ignore UAV AI",typeOf _unit);};
+        if (getNumber (configOf _unit >> "isPlayableLogic") == 1) exitWith {TRACE_1("ignore logic unit",typeOf _unit);};
 
+        private _allHitPoints = getAllHitPointsDamage _unit param [0, []];
+        if ((GVAR(customHitpoints) arrayIntersect _allHitPoints) isNotEqualTo GVAR(customHitpoints)) exitWith {
+            ERROR_1("Bad hitpoints for unit type ""%1""",typeOf _unit);
+        };
 
-// this handles moving units into vehicles via load functions or zeus
-// needed, because the vanilla INCAPACITATED state does not handle vehicles
-["CAManBase", "GetInMan", {
-    params ["_unit", "", "_vehicle"];
+        // Calling this function inside curly brackets allows the usage of
+        // "exitWith", which would be broken with "HandleDamage" otherwise.
+        _unit setVariable [
+            QEGVAR(medical,HandleDamageEHID),
+            _unit addEventHandler ["HandleDamage", {_this call FUNC(handleDamage)}]
+        ];
 
-    if (local _unit && {lifeState _unit == "INCAPACITATED"}) then {
-        [_unit, true] call FUNC(setUnconsciousAnim);
-    };
+        _unit addEventHandler ["HandleHeal", {
```

Comment:
Need to check with MP testing how to actually behaves. Wiki says it fires for local units only (like `HandleDamage`).

Reply:
Fires where healer is local (as wiki says).